pkg/netif/netif.go

```diff
@@ -23,7 +27,9 @@ func (m *NetifManager) EnsureDummyDevice(name string) (bool, error) {
 	l, err := m.LinkByName(name)
 	if err == nil {
 		// found dummy device, make sure ip matches. AddrAdd will return error if address exists, will add it otherwise
-		m.AddrAdd(l, m.Addr)
+		for _, addr := range m.Addrs {
+			m.AddrAdd(l, addr)
```

Comment:
Check error here?

Reply:
AddrAdd will return error if the address already exists. That is a benign error, hence I did not include checking here. EnsureDummyDevice is called periodically. We do check the error in AddDummyDevice for the AddrAdd calls.